site/content/docs/02-routing.md

```diff
@@ -118,6 +118,25 @@ The `error` object is made available to the template along with the HTTP `status
 
 
 
+### Error handling for server routes
+
+By default, errors occurred in server routes are sent as a text response with the HTTP 500 status. You can define your own error handler by passing it as an option to Sapper middleware:
+
+```js
+function on_error(err, req, res, next) {
+  res.setHeader('Content-Type', 'application/json');
```

Comment:
the formatting here is inconsistent. the spaces on this line and the next should replaced with tabs for consistency with the rest of the file